sentry-android-ndk/src/main/java/io/sentry/android/ndk/SentryNdk.java

```diff
@@ -16,7 +16,8 @@ private SentryNdk() {}
     // See
     // https://android.googlesource.com/platform/bionic/+/master/android-changes-for-ndk-developers.md#changes-to-library-dependency-resolution
     System.loadLibrary("log");
-    System.loadLibrary("sentry");
+    // looks like sentry is built as a static lib and the final 'so' is not generated
+//    System.loadLibrary("sentry");
```

Comment:
@Swatinem

Reply:
@philipphofmann that's why your release missed the sentry.so file and failed to capture events, I'm reaching out to the native team so see what have changed